.gitlab/ci/sdk-nightly.yml

```diff
@@ -155,6 +155,22 @@ demisto-sdk-nightly:run-commands-against-instance:
     - section_end "Destroy instances"
 
 
+demisto-sdk-nightly:run-end-to-end-tests:
+  tags:
+    - gke # Better when not needing docker-in-docker
+  extends:
+    - .sdk-nightly-schedule-rule-always
```

Comment:
Do we need `.default-job-settings` too?

Reply:
Same as `demisto-sdk-nightly:run-commands-against-instance`